test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -381,10 +381,10 @@ TEST_F(ClusterManagerImplTest, OriginalDstLbRestriction2) {
                   port_value: 11001
   )EOF";
 
-  EXPECT_THROW_WITH_MESSAGE(
-      create(parseBootstrapFromV2Yaml(yaml)), EnvoyException,
-      "cluster: LB policy ORIGINAL_DST_LB is not valid for Cluster type STATIC. "
-      "'ORIGINAL_DST_LB' is allowed only with cluster type 'ORIGINAL_DST'");
+  EXPECT_THROW_WITH_MESSAGE(create(parseBootstrapFromV2Yaml(yaml)), EnvoyException,
```

Comment:
shall we rename `parseBootstrapFromV2Yaml` to `parseBootstrapFromV3AlphaYaml`?

Reply:
I'll take an AI to do the cleanup in a follow PR, ideally we aim to keep this one as small as possible given the large mechanical base.

Reply:
(there are other methods to deal with here)